optimize/util/optimize-commons/pom.xml

```diff
@@ -134,22 +142,26 @@
       <artifactId>commons-lang3</artifactId>
       <version>3.17.0</version>
     </dependency>
+    <!-- unused/declared
     <dependency>
       <groupId>com.ethlo.time</groupId>
       <artifactId>itu</artifactId>
       <version>1.10.2</version>
     </dependency>
```

Comment:
I think this is used but it's in a commented out test. This scenario makes things a little more difficult because it's also probably true for a bunch of other tests

Reply:
I think we can still leave them commented out with a TODO.
Once we re-enable the tests, they would stop the compilation if the dependency is not present, so the problem would be visible

Reply:
I think we are more likely to delete the tests than reenable them, and build again from scratch. The delta between Optimize 7 and 8 ITs is quiet great, and reenabling/fixing them is difficult